__tests__/app.test.js

```diff
@@ -117,7 +117,6 @@ describe("GET /api/articles/:article_id/comments", () => {
           body: expect.any(String),
           votes: expect.any(Number),
           author: expect.any(String),
-          article_id: expect.any(Number),
```

Comment:
You could have just put the correct id here and your matcher in the forEach would have still worked.